### DISCUSSION TITLE
Customize log messages for user storage LDAP configuration in KC shown in admin UI

### DISCUSSION THREAD

**Discussion by souravs17031999** [keycloak/keycloak#15416] [Q&A]
Hi team, 

I have setup of LDAP active directory configured with KC and as a end user perspective, I want to make sure whenever this configuration is wrongly configured, I get clear and specific message in admin console UI to help in troubleshooting.

What is currently happening is that Keycloak is throwing generic error messages in UI although application (deployed as docker container) itself shows some logs which are helpful but those are all in backend and doesn't helps end-users who are exposed only to admin UI.

Is it possible somehow to modify those log messages in the UI and get specific messages as per the exact mis-configuration ?

Note: Even the API response is generic message "unknown_error", "LDAP test error" etc... not indicating the exact issue.

I will attach one such scenario for better understanding here.
Ex. below, It would be nice if it shows the unknown host/wrong host configured issue.

[keycloak LDAP configuration]
![paint](https://user-images.githubusercontent.com/33771969/200897797-47531052-6e5d-4e9f-8907-b6558aaf393c.PNG)

[keycloak container logs]
![keycloak-backend](https://user-images.githubusercontent.com/33771969/200762154-d732e693-f843-4843-8db9-2092c45de7ea.PNG)

[keycloak UI]
![keycloak-admin](https://user-images.githubusercontent.com/33771969/200763266-39e7123e-adea-4230-88ba-3bddfea236c5.PNG)

**Comment by chiragbarco** [comment]
Yes, I feel it is a great convenience to get pin-point information on the field value(s) that is(are) causing errors. This saves time, energy and need to revisit/recheck and assume configurations that might not be working. Hence, I too request for more clear and non-ambiguous feedbacks.
